scripts/build_reproducibility_index

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Check that artifacts can be reproducibly built, both when re-building them on the same machine
+# over time, but also when built on different machines (e.g. locally vs on GCP).
+#
+# This script should be re-run every time any of these artifacts changes, and should be checked in
+# so that we can enforce in CI that the hashes are indeed consistent.
+
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+# List of artifacts that are expected to be reproducibly built.
+readonly REPRODUCIBLE_ARTIFACTS=(
+  ./target/wasm32-unknown-unknown/release/*.wasm
+  ./bazel-clang-bin/oak/server/loader/oak_runner
+  ./target/debug/oak_loader
```

Comment:
Maybe it's better to use `release` than `debug` ?

Reply:
AFAICT that's not what is built / used by the other scripts?

https://github.com/project-oak/oak/blob/55a93d7b7f3a23687db42d152d51eaf69a1fc234/scripts/build_server#L50

We should probably switch that to `--release`, at which point we can change this list too.